manim/mobject/graphing/coordinate_systems.py

```diff
@@ -575,29 +577,32 @@ def construct(self):
                     line = ax.get_horizontal_line(point, line_func=Line)
 
                     self.add(ax, line, dot)
+        """
+
+        return self.get_line_from_axis_to_point(1, point, **kwargs)
+
+    def get_lines_to_point(self, point: Sequence[float], **kwargs) -> VGroup:
+        """Generate both horizontal and vertical lines from the axis to a point.
 
         Parameters
         ----------
         point
-            The point to which the horizontal line will be drawn.
-
+            A point on the scene.
         kwargs
-            Additional parameters to be passed to :class:`get_line_from_axis_to_point`
+            Additional arguments to be provided :meth:`CoordinateSystem.get_line_from_axis_to_point`.
```

Comment:
I think I prefer the previous wording better, 🤔

Reply:
I don't like it either and neither do I remember ever doing such a change 🤔